src/java/org/apache/cassandra/tools/nodetool/GuardrailsConfigCommand.java

```diff
@@ -59,51 +61,101 @@ public static class GetGuardrailsConfig extends GuardrailsConfigCommand
         allowedValues = { "values", "thresholds", "flags", "others" })
         private String guardrailCategory;
 
-        @Arguments(description = "Specific names or guardrails to get configuration of.")
+        @Option(name = { "--verbose", "-v" })
+        private boolean verbose = false;
+
+        @Arguments(description = "Specific name of a guardrail to get configuration of.")
```

Comment:
we can narrow it down to a single string argument:

```suggestion
        @Arguments(description = "Specific name of a guardrail to get configuration of.")
        private String guardrailName;
```